README.md

```diff
@@ -125,6 +144,92 @@ https://github.com/kbase/user_profile/blob/master/UserProfile.spec
   ]
 }
 ```
+</details>
+
+## @app.route('/fetchUserProfiles')
+
+### Synopsis:
+When this route recieves a POST request with a JSON body from a JS object like
+```js
+{ users: [...userIDs] }
+```
+this route will return an array of the profiles for each user, and `null` if a
```

Comment:
Maybe make this a bit clearer, like the upstream docs:
Given a list of usernames, returns a list of UserProfiles in the same order.
If no UserProfile was found for a username, the UserProfile at that position will be null.